.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "env": {
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
```

Comment:
`Atomics` and `SharedArrayBuffer` are [native objects in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics), like `window` when in a browser.

These rules enforces to not assign another value to these objects.

![image](https://user-images.githubusercontent.com/6919329/66443872-353f5080-ea17-11e9-80d7-590d33d87476.png)